src/main/java/com/agricraft/agricore/lang/AgriString.java

```diff
@@ -4,27 +4,51 @@
 import com.google.gson.annotations.SerializedName;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.TreeMap;
 
 public class AgriString {
 
-    private final Map<String, String> translations;
-
     @SerializedName(value = "default", alternate = {"normal"})
     private final String normal;
+    private final Map<String, String> translations;
 
     public AgriString() {
+        this.normal = "add translations in here.";
         this.translations = new HashMap<>();
-        this.normal = "Testing 1, 2, 3.";
+        this.translations.put("en_US", normal);
     }
 
     public AgriString(String value) {
-        this.translations = new HashMap<>(0);
         this.normal = value;
+        this.translations = new HashMap<>();
+        this.translations.put("en_US", normal);
+    }
+
+    public AgriString(String normal, Map<String, String> translations) {
+        this.normal = normal;
+        this.translations = translations;
+        if (this.translations.isEmpty()) {
+            this.translations.put("en_US", normal);
+        }
+
+    }
+
+    public String getNormal() {
+        return normal;
+    }
+
+    public TreeMap<String, String> getTranslations() {
+        TreeMap<String, String> result = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+        result.putAll(translations);
+        return result;
     }
 
     @Override
     public String toString() {
-        return this.translations.getOrDefault(AgriCore.getTranslator().getLocale(), normal);
+        if (getTranslations().containsKey(AgriCore.getTranslator().getLocale())){
+            return getTranslations().get(AgriCore.getTranslator().getLocale());
+        }
+        return normal;
```

Comment:
It's been a while since I last used java, is there some sort of performance issue with using `getOrDefault()` that means we can't use it here?

Reply:
We can definitely use that here.